package.json

```diff
@@ -24,24 +24,24 @@
   "homepage": "https://github.com/amannn/action-semantic-pull-request#readme",
   "dependencies": {
     "@actions/core": "^1.10.0",
-    "@actions/github": "^5.0.0",
-    "conventional-changelog-conventionalcommits": "4.5.0",
+    "@actions/github": "^5.1.1",
+    "conventional-changelog-conventionalcommits": "5.0.0",
     "conventional-commit-types": "^3.0.0",
     "conventional-commits-parser": "^3.2.4"
   },
   "devDependencies": {
-    "@semantic-release/changelog": "5.0.1",
-    "@semantic-release/commit-analyzer": "8.0.1",
-    "@semantic-release/git": "9.0.0",
-    "@semantic-release/github": "7.2.0",
-    "@semantic-release/release-notes-generator": "9.0.3",
-    "@vercel/ncc": "^0.27.0",
-    "eslint": "7.9.0",
-    "eslint-config-molindo": "5.1.1",
-    "jest": "27.4.7",
-    "semantic-release": "19.0.2"
+    "@semantic-release/changelog": "6.0.2",
+    "@semantic-release/commit-analyzer": "9.0.2",
+    "@semantic-release/git": "10.0.1",
+    "@semantic-release/github": "8.0.7",
+    "@semantic-release/release-notes-generator": "10.0.3",
+    "@vercel/ncc": "^0.36.1",
+    "eslint": "8.36.0",
+    "eslint-config-molindo": "6.0.0",
+    "jest": "29.5.0",
+    "semantic-release": "20.1.1"
   },
   "engines": {
-    "node": "^16.0.0"
+    "node": "^18.0.0"
```

Comment:
Since `@semantic-release` is only a dev dependency, we could probably express this requirement with `devEngines` and not have a breaking change—right?

Reply:
Sure, can check that. Didn't think of that. I partly thought of this, due to node 18 already being the current LTS. Therefore I was more inclined to update to the current LTS.

Reply:
I tried to add `devEngines`, but the system won't accept the options to have either `devEngines` seperately or have both options there.
![image](https://user-images.githubusercontent.com/5102501/225253272-5d28feda-615f-47e4-93e6-e8edc407d165.png)

Reply:
Oh sorry, I just noticed I had a flaw in my logic 🤦‍♂️. We can of course bump the `engines`, since the consumer doesn't pick the Node.js version. Instead we should bump it here:

https://github.com/amannn/action-semantic-pull-request/blob/b6bca70dcd3e56e896605356ce09b76f7e1e0d39/action.yml#L5

Sorry for the back and forth!

Reply:
ah, ok. I've been searching for this for a bit now. There seems to be an issue with `- using` atm. There is a workaround (https://github.com/actions/runner-images/issues/7002#issuecomment-1442874916) I'll implement on the action

Reply:
You mean `node18` doesn't work out of the box? Is the workaround to use `node16` and install Node.js 18 on top? Does that impact the run time much?

Reply:
> You mean `node18` doesn't work out of the box? Is the workaround to use `node16` and install Node.js 18 on top? Does that impact the run time much?

No it does not.
I tried a workaround which I mentioned earlier. This does not seem to work either, as i'm getting the following error:
> Error: /home/runner/work/Brink.Github.Actions.SemanticPullRequest/Brink.Github.Actions.SemanticPullRequest/./action.yml (Line: 7, Col: 13): Expected format {org}/{repo}[/path]@ref. Actual 'actions/setup-node'

As I'm diving in this in more detail, I'm looking at the logs of the last working version (https://github.com/Brink-Software/Brink.Github.Actions.SemanticPullRequest/actions/runs/4424327485/jobs/7758018566).
There, I stated in the setup files this should be using 
```
- uses: actions/setup-node@v2
        with:
          node-version: 18
```
however the logging states 
```
Run actions/setup-node@v[2](https://github.com/Brink-Software/Brink.Github.Actions.SemanticPullRequest/actions/runs/4424327485/jobs/7758018566#step:3:2)
  with:
    node-version: 16
    always-auth: false
    check-latest: false
    token: ***
Found in cache @ /opt/hostedtoolcache/node/16.19.1/x6[4](https://github.com/Brink-Software/Brink.Github.Actions.SemanticPullRequest/actions/runs/4424327485/jobs/7758018566#step:3:4)
```
So I'm pondering what to do.
My suggestion would be, as seems to be the case: that building with node18 seems to work for `semantic-release` and the action seems to run at node 16. I'm baffled by the outcome, because previous checks didn't work with version 16... 
I'll try that, so workflow files might not have to be adapted. 
If it succeeds, it's fine
If it doesn't, I guess it wants the v18, where it's just not used on the action run itself 🤔. I would proceed with the current set of commits then
What's your opinion?

Reply:
So, with the last run, it _did_ update node to 18. The action runs fine. 
However, there is no support for moving the action.yml to v18... 

I would suggest making it an gh issue for future resolving, as it does seem to work with both node 16 and 18, if the runs are anything to go by

Reply:
Ok, So I've come up with a compromise.
Since `semantic-release` is the only thing needing node18, the actual **running** can be done on node16, making it a less to non-breaking change?

Reply:
Hmm, I have to say, this seems to introduce more complexity than what I'd have hoped for in regards to a PR that updates dependencies. Downgrading the Node.js version after an install seems a bit like a hack to me and it could potentially cause troubles.

It seems like quite a bold move of `semantic-release` to already drop Node.js 16 at this point, given that this doesn't work yet in GitHub Actions and the version is officially maintained.

I would suggest to exclude the `semantic-release` upgrade from this PR at this point and waiting for GitHub Actions to support Node.js 18 properly before moving forward with this. We could update `semantic-release` to the latest 19 minor version though.

What do you think?

Reply:
Thank you for your patience btw., I really appreciate the work you're putting into this PR! 🙏

Reply:
> Thank you for your patience btw., I really appreciate the work you're putting into this PR! 🙏

Thank you for the kind words. To me, it is my first _real_ endeavor giving to the open source community 😊



> Hmm, I have to say, this seems to introduce more complexity than what I'd have hoped for in regards to a PR that updates dependencies. Downgrading the Node.js version after an install seems a bit like a hack to me and it could potentially cause troubles.
> 
> It seems like quite a bold move of `semantic-release` to already drop Node.js 16 at this point, given that this doesn't work yet in GitHub Actions and the version is officially maintained.
> 
> I would suggest to exclude the `semantic-release` upgrade from this PR at this point and waiting for GitHub Actions to support Node.js 18 properly before moving forward with this. We could update `semantic-release` to the latest 19 minor version though.
> 
> What do you think?

I would agree on this. I updated the PR to reflect this

Reply:
> To me, it is my first _real_ endeavor giving to the open source community 😊. Any feedback/A shoutout at release would be nice 😀.

That's fantastic, keep up the great work! 🙌 

You're listed here as the contributor of this release: https://github.com/amannn/action-semantic-pull-request/releases/tag/v5.2.0